### ISSUE TITLE
Using conan or vcpgk to resolve thirdparty dependencies.

### ISSUE THREAD

**Issue by matrixji** [milvus-io/milvus-sdk-cpp#218]
Now we have to use External Project in cmake to clone the code from GitHub, there will be two issues:
- The initial download size is too large
- It's difficult to build while with a limited network.

Try using conan2(prefer) or vcpkg to manage the dependencies.

**Comment by xiaofan-luan** [comment]
> Now we have to use External Project in cmake to clone the code from GitHub, there will be two issues:
> 
> * The initial download size is too large
> * It's difficult to build while with a limited network.
> 
> Try using conan2(prefer) or vcpkg to manage the dependencies.

Milvus already use conan. Feel free to discuss with @jiaoew1991

**Comment by matrixji** [comment]
> Milvus already use conan. Feel free to discuss with @jiaoew1991
Yes, I see that on master, so I'd like to see conan is the 1st choice for cpp sdk at the moment

**Comment by yhmo** [comment]
Already support conan 2.x from v2.6.
